docs: describe how to remove extensions from the private openvsx registry (#3029)
codenvy/che-docs: modules/administration-guide/partials/proc_deleting-extensions-in-private-openvsx-registry.adoc

> You can remove extensions from your private OpenVSX registry in two ways: by using the OpenVSX Admin API (recommended) or by deleting the extension data directly in the PostgreSQL database.

Reviewer: ```suggestion
You can remove extensions from your private OpenVSX registry by using one of the following methods:

* Use the OpenVSX administrator API (recommended).
* Delete the extension data directly in the PostgreSQL database.
```

[id="delete-extensions-admin-api"]

> === Option 1: Delete extension using the OpenVSX Admin API

Reviewer: ```suggestion
== Deleting an extension by using the OpenVSX administrator API
```

> This method uses an admin user and Personal Access Token (PAT) to call the Admin API.

Reviewer: ```suggestion
Use this method to call the administrator API with an administrator user and a personal access token (PAT).
```

.Prerequisites

> * Access to the OpenShift cluster where OpenVSX is deployed (namespace `openvsx`).

Reviewer: ```suggestion
* You must have access to the OpenShift cluster where OpenVSX is deployed (namespace `openvsx`).
```

.Procedure

> . Add the OpenVSX admin user and PAT to the database.

Reviewer: ```suggestion
. Add the OpenVSX administrator user and PAT to the database:
```

+
Get the PostgreSQL pod name:
+
[bash,subs="verbatim",options="nowrap"]
----
export POSTGRESQL_POD_NAME=$(oc get pods -n openvsx \
   -o jsonpath="{.items[*].metadata.name}" | tr ' ' '\n' | grep '^postgresql' | head -n 1)
----
+
Add the admin user:
+
[bash,subs="verbatim",options="nowrap"]
----
oc exec -n openvsx "$POSTGRESQL_POD_NAME" -- bash -c \
   "psql -d openvsx -c \"INSERT INTO user_data (id, login_name, role) VALUES (1002, 'openvsx-admin', 'admin');\""
----
+
Add the admin PAT:
+
[bash,subs="verbatim",options="nowrap"]
----
oc exec -n openvsx "$POSTGRESQL_POD_NAME" -- bash -c \
   "psql -d openvsx -c \"INSERT INTO personal_access_token (id, user_data, value, active, created_timestamp, accessed_timestamp, description) VALUES (1002, 1002, 'openvsx_admin_token', true, current_timestamp, current_timestamp, 'Admin API Token');\""
----
+
[NOTE]
====

> Use a strong, unique value instead of `openvsx_admin_token` for production. The PAT is passed in the API request URL.

Reviewer: ```suggestion
Use a strong, unique value instead of `openvsx_admin_token` for production. The system passes the PAT in the API request URL.
```
